tuning/utils/import_utils.py

```diff
@@ -12,11 +12,14 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# Standard
+import os
+
 # Third Party
 from transformers.utils.import_utils import _is_package_available
 
 _is_aim_available = _is_package_available("aim")
 
 
 def is_aim_available():
-    return _is_aim_available
+    return _is_aim_available and (os.environ.get("AIMSTACK_DB") is not None)
```

Comment:
@HarikrishnanBalagopal Aim installation and setup does not depend only on the AIMSTACK_DB it depends on server as well. I think its best to keep this change separate and I can have a separate PR for this fix as this leaves the check incomplete.

Reply:
```
I think its best to keep this change separate and I can have a separate PR for this fix as this leaves the check incomplete.
```

We had to fix this bug just for testing this PR that's why it's part of it.

```
@HarikrishnanBalagopal Aim installation and setup does not depend only on the AIMSTACK_DB it depends on server as well. 
```

Yes we can add extra checks, @kmehant suggested checking for `AIMSTACK_DB`. We can check for all 3 environment variables as well.

Reply:
@dushyantbehl We were debugging end to end tests and had to add this check tentatively using `AIMSTACK_DB` env if it has to be `AIMSTACK_SERVER` hari will update this PR. 

Please feel free to open another PR if there are additional checks that has to go into this.

Reply:
@dushyantbehl @kmehant I have separated the fix into a new PR.